msprime/trees.py

```diff
@@ -77,6 +77,8 @@ def __repr__(self):
 class Individual(SimpleContainer):
     """
     An :ref:`individual <sec_individual_table_definition>` in a tree sequence.
+    Since nodes correspond to genomes, individuals are associated with a collection
+    of nodes (e.g., two nodes per diploid).
```

Comment:
this is a hard thing to explain concisely! just a reminder here, really.

Reply:
Maybe refer back to the "nodes, genomes and individuals" section for more details?